lib/partition_setup.pm

```diff
@@ -457,6 +559,11 @@ sub take_first_disk_storage_ng {
     }
 }
 
+=head2 take_first_disk
+
+ take_first_disk([iscsi => $iscsi]);
```

Comment:
Missing description here.

Reply:
I have honestly no clue what it is doing exactly, of course I can write "takes first disk" but that would add no value